cmd/thanos/example.go

```diff
@@ -60,12 +53,6 @@ func registerExample(m map[string]setupFunc, app *kingpin.Application, name stri
 		}
 		g.AddGroup(sidecarGroup)
 
-		storeGroup, err := runStore(logger, metrics, *gcsBucket, *maxDiskCacheSize, *maxMemCacheSize)
```

Comment:
This is temporary or we don't want store in example dir?

Reply:
I could've made it compile but it wouldn't have worked regardless. Figured I just drop it and we just revisit the command properly later.